doc/source/whatsnew/v0.23.0.txt

```diff
@@ -423,6 +423,9 @@ Conversion
 - Bug in :class:`WeekOfMonth` and :class:`LastWeekOfMonth` where default keyword arguments for constructor raised ``ValueError`` (:issue:`19142`)
 - Bug in localization of a naive, datetime string in a ``Series`` constructor with a ``datetime64[ns, tz]`` dtype (:issue:`174151`)
 - :func:`Timestamp.replace` will now handle Daylight Savings transitions gracefully (:issue:`18319`)
+- Bug in :class:`Index` multiplication and division methods where operating with a ``Series`` would return an ``Index`` object instead of a ``Series`` object (:issue:`19042`)
```

Comment:
in another PR, pls separate out all of the ops changes into a separate sub-section of Bug Fixes (as I think we have a lot), call it Operations

Reply:
Sure.  All 4 current entries in the Numeric subsection could plausibly belong in Operations.  Move them or leave them be?

Reply:
Also a) also move comparison method changes or just arithmetic methods and b) also move offset arithmetic, or just Series/Index?